zipline/api.pyi

```diff
@@ -545,6 +545,8 @@ def schedule_function(func, date_rule=None, time_rule=None, half_days=True, cale
         The rule for the times to execute this function.
     half_days : bool, optional
         Should this rule fire on half days?
+    calendar : Sentinel, optional
```

Comment:
Yea. It certainly doesn't hurt anything, but want you to know it's not required.